### DISCUSSION TITLE
Why are actions down on a weekly basis ?

### DISCUSSION THREAD

**Discussion by maximelebastard** [community/community#55069] [Actions] [answered]
### Select Topic Area

Product Feedback

### Body

Hi,

It has become a weekly routine: Actions are down again this Wednesday.
For us who rely on this service to ship software, a +4 hours downtime is very disturbing.

Actions have been down 9 times since April 1st, often for a few hours everytime.
Do you have any blogpost explaining the cause of these frequent incidents and are you planning to do something to reduce the impact of those failures on your customers ?

**Comment by jmillsy** [comment, marked as answer]
Hey  @maximelebastard 

First, I want to apologize for the outages that we’ve had recently. I understand how much those incidents can impact you and your day to day productivity. Improving our availability and reliability is our primary focus, and we are actively investing in both short term and long term ways to improve Actions reliability. For a deeper understanding of the recent outages, we published a [blog](https://github.blog/2023-05-16-addressing-githubs-recent-availability-issues/) post that goes into more detail. Thank you for sharing your feedback - we truly appreciate it.

**Reply by DirkLachowski** [reply]
That didn't age too well.

**Reply by cryptic-key** [reply]
Crapped out again

**Reply by wvanrensburg** [reply]
If you hold several Security certificates, such as SOC 2 etc, then why do you allow unmonitored, untested configuration changes to your environment? Seems like you are breaking protocol here.

**Reply by clong-viimed** [reply]
Also the SLA is being breached too and our company has yet to see any kind of compensation for lost developer time, lost AI tokens usage because of agent's dieing in the middle of output. etc...,

**Comment by cryptic-mystic** [comment]
Here in 2026, and this is still an issue. Outages pretty much weekly. Pretty abysmal honestly.

**Comment by dongraMW** [comment]
Every single week a core service is down. On top of that, the intentionally misleading and downplaying in the status pages is disappointing. There should be a user notification when GH services are actually functional so we know which 2 hours each week we can actually rely on GH.

**Comment by mikebollandajw** [comment]
down again - useless

**Reply by DirkLachowski** [reply]
Yep, Any recommendation for a replacement for Actions besides self hosting GiltLab? Would prefer something managed but with a decent uptime. GH Actions have effectively become a dead weight.

**Reply by mikebollandajw** [reply]
problem is even webhooks was down :(

**Comment by DirkLachowski** [comment]
Wow, now Actions being down has crashed the community. Can't make up this shit. 

<img width="1092" height="869" alt="image" src="https://github.com/user-attachments/assets/640a8156-f1eb-46c6-a442-ea25dcd24205" />
